src/OSPSuite.Presentation/Mappers/ParameterIdentifications/ParameterIdentificationRunResultToRunResultDTOMapper.cs

```diff
@@ -24,35 +24,50 @@ public ParameterIdentificationRunResultToRunResultDTOMapper(IOptimizedParameterR
       public ParameterIdentificationRunResultDTO MapFrom(ParameterIdentification parameterIdentification, ParameterIdentificationRunResult runResult)
       {
          var runResultDTO = new ParameterIdentificationRunResultDTO(runResult);
-        
+
          runResult.BestResult.Values.Each(x =>
          {
             var optimizedParameterDTO = mapFrom(x, parameterIdentification);
-            if (optimizedParameterDTO != null)
-            {
-               runResultDTO.AddOptimizedParameter(optimizedParameterDTO);
-               if (runResultDTO.LegendImage == null)
-                  runResultDTO.LegendImage = _optimizedParameterRangeImageCreator.CreateLegendFor(optimizedParameterDTO);
-            }
+            addOptimizedParameterDTOTo(runResultDTO, optimizedParameterDTO);
+         });
 
+         parameterIdentification.AllFixedIdentificationParameters.Each(x =>
```

Comment:
I am adding the fixed parameter to the results so that the user can see for example WHAT is going to be transferred to the simulation (would be too much of a pain to have to always click on the configure view to see the actual fixed value).